docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -18,100 +18,168 @@ Within the concept of replication, universes are divided into the following cate
 
-- Configure the replication, as follows:
+  - To delete a table from the replication, click **... > Remove Table**. This removes both the table and its index tables from replication. Note that you may prevent an index table from being replicated without affecting its main table.
 
-  - Click **Actions > Edit replication configuration**.
+  - To add more tables to the replication, click **Add Tables** to open the **Add Tables to Replication** dialog which allows you to make a selection from a list of tables that have not been replicated, and then click **Validate Table Selection**.
 
-  - Make changes using the **Edit cluster replication** dialog shown in the following illustration:<br>
+    Note that a table with the same name in the same database (or the same namespace) with the same schema must exist on the target universe before the table can be added. Otherwise the table is tagged as No Match and cannot be selected. See [About the table selection](#about-the-table-selection) for additional information.
+    
+    When using YCQL, it is recommended to add new tables as soon as they are created. If the table contains enough data, and then you decide to add it to the replication configuration, bootstrapping will be required, in which case other tables in the same database must be removed from the replication and also subjected to bootstrapping.
+    
+    When using YSQL, new tables in a database can be added to the replication configuration. Moreover, if a subset of YSQL tables of a database is already in replication in a configuration, the remaining tables in that database must be added to the same configuration.
 
-    ![Edit Replication](/images/yp/asynch-replication-5.png)<br>
+- View detailed statistics on the asynchronous replication lag by selecting **Metrics**, as per the following illustration:
 
-- Set up monitoring by configuring alerts, as follows:
+  ![Alert](/images/yp/asynch-replication-61.png)<br>
 
-  - Click **Configure Alert**.
+## Set up bidirectional replication
 
-  - Use the **Configure Replication Alert** dialog to enable or disable alert issued when the replication lag exceeds the specified threshold, as per the following illustration:<br>
+You can set up bidirectional replication using either the YugabyteDB Anywhere UI or API by creating two separate replication configurations. Under this scenario, a source universe of the first replication becomes the target universe of the second replication, and vice versa.
 
-    ![Alert](/images/yp/asynch-replication-6.png)<br>
+Bootstrapping is not enabled for bidirectional replication.
```

Comment:
[Please rephrase it before putting it in the doc]
Bootstrapping will be skipped for tables on the target universe that are already in replication. That means while setting up bidirectional replication, bootstrapping can be done in only one direction, the first replication config. For example, when you set up replication for table t1 from universe u1 to u2, it may go through bootstrapping. However, if you set up replication, in the reverse direction, from universe u2 to u1 for table t1, bootstrapping will be skipped.

Reply:
Added